react/SelectBox/ControlDefault.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { components } from 'react-select'
+
+const ControlDefault = ({ selectProps: { onControlClicked }, ...props }) => {
+  // onTouchStart is necessary on mobile
+  // see https://github.com/JedWatson/react-select/issues/3806#issuecomment-541325710
+  return (
+    <div
+      onClick={onControlClicked}
+      onTouchStart={onControlClicked}
+      data-testid="selectBox-controlDefault"
+    >
+      <components.Control {...props} />
```

Comment:
Why not setting the onClick / onTouchStart on the `components.Control`? We're adding a new `div` which creates a new node. Can't we avoid that?

Reply:
No we can't, passing props (onClick, onTouchStart, data-testid) directly on <components.Control {...props} /> won't work

Reply:
And something like: 

```
const ControlDefault = ({ selectProps: { onControlClicked }, innerProps, ...props }) => {

const enhancedInnerProps = {...innerProps, onClick: onControlClicked, onTouchStart: onControlClicked} 

<components.Control {...props} innerProps={enhancedInnerProps} />
```

is not working? 🤔

Reply:
yes it does, thanks 👍

Reply:
Next time, take a look at the `react-select` implementation to understand why passing `props` was not working (https://github.com/JedWatson/react-select/blob/v2.x/src/components/Control.js#L56)